pkgs/by-name/sa/saman_ttf/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+}:
+
+stdenvNoCC.mkDerivation rec {
```

Comment:
`finalAttrs` is usually preferred over `rec` because [it works better with overriding](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)